src/coloring.jl

```diff
@@ -642,15 +645,29 @@ function postprocess!(
 
     # if at least one of the colors is useless, modify the color assignments of vertices
     if any(!, color_used)
-        # assign the neutral color to every vertex with a useless color
+        num_colors_useless = 0
+
+        # determine what are the useless colors and compute the offsets
+        for i in 1:nb_colors
+            if color_used[i]
+                offsets[i] = num_colors_useless
+            else
+                num_colors_useless += 1
+                offsets[i] = -num_colors_useless
```

Comment:
```suggestion
```

Reply:
We don't need to compute the `offset` for neutral colors because we don't use it.